consensus/types/src/beacon_state.rs

```diff
@@ -134,7 +147,21 @@ pub enum Error {
         current_epoch: Epoch,
         epoch: Epoch,
     },
+    CommitteeCacheDiffInvalidEpoch {
+        prev_current_epoch: Epoch,
+        current_epoch: Epoch,
+    },
+    CommitteeCacheDiffUninitialized {
+        expected_epoch: Epoch,
+    },
+    DiffAcrossFork {
+        prev_fork: ForkName,
+        current_fork: ForkName,
+    },
+    TotalActiveBalanceDiffUninitialized,
+    MissingImmutableValidator(usize),
```

Comment:
Where are these used? I can't find usage on the current branch, i guess they're on tree-states?

Reply:
I think they may be leftover and unused on tree-states too! Will delete